test/RequestTest.php

```diff
@@ -120,7 +120,7 @@ public function invalidRequestMethod()
             'false'      => [ false ],
             'int'        => [ 1 ],
             'float'      => [ 1.1 ],
-            'bad-string' => [ 'BOGUS-METHOD' ],
+            'bad-string' => [ 'BOGUS METHOD' ],
```

Comment:
This test case had to change as the string was a valid token! Adding whitespace invalidated it again.